chainercv/datasets/pascal_voc/voc_detection_dataset.py

```diff
@@ -61,6 +61,8 @@ class VOCDetectionDataset(chainer.dataset.DatasetMixin):
         return_difficult (bool): If true, this dataset returns a boolean array
             that indicates whether bounding boxes are labeled as difficult
             or not. The default value is :obj:`False`.
+            If :obj:`use_difficult` is :obj:`False`, this has to be
+            :obj:`False`.
```

Comment:
How about returning a false array as `difficult` when `use_difficult=False and return_difficult=True`?